pygmt/figure.py

```diff
@@ -257,12 +257,19 @@ def savefig(
         """
         Save the figure to a file.
 
-        This method implements a matplotlib-like interface for
-        :meth:`pygmt.Figure.psconvert`.
+        Supported file formats and their extensions:
 
-        Supported formats: PNG (``.png``), JPEG (``.jpg`` or ``.jpeg``),
-        PDF (``.pdf``), BMP (``.bmp``), TIFF (``.tif``), EPS (``.eps``), and
-        KML (``.kml``). The KML output generates a companion PNG file.
+        - PNG (``.png``)
+        - JPEG (``.jpg`` or ``.jpeg``)
+        - PDF (``.pdf``)
+        - BMP (``.bmp``)
+        - TIFF (``.tif`` or ``.tiff``)
+        - EPS (``.eps``)
+        - KML (``.kml``)
+
+        For TIFF format, ``.tiff`` generates a GeoTIFF file with embedded
+        georeferencing information and a companion world file. For KML format,
```

Comment:
Maybe clearer to split into two lines?

```suggestion
        - TIFF (``.tif``)
        - GeoTIFF (``.tiff``)
        - EPS (``.eps``)
        - KML (``.kml``)

        For TIFF format, ``.tiff`` generates a GeoTIFF file with embedded
        georeferencing information and a companion world file. For KML format,
```